src/upload/UploadPanel.stories.tsx

```diff
@@ -58,6 +60,22 @@ export const empty = () => <UploadPanel
     onDoneAll={action("doneAll")}
 />;
 
+empty.story = {
+    parameters: {
+        async puppeteerTest(page: any) {
+            const filesPath = relative(process.cwd(), __dirname);
+            const uploadAddedActionPromise = interceptAction(page, "uploadAdded");
+
+            const uploadInput = await page.$("input");
+            await uploadInput.uploadFile(filesPath + "/testFile1");
+            await uploadInput.evaluate((input: HTMLInputElement) => input.dispatchEvent(new Event("change", { bubbles: true })));
+            expect(uploadAddedActionPromise).resolves;
```

Comment:
Shouldn't it be like this? Usually expect(ations)/assert(ions) are functions instead of properties.
```suggestion
            expect(uploadAddedActionPromise).resolves();
```

Reply:
It is a property but actually I wasn't going to use it since it gave false positives. I only noticed that in the next story, so I forgot to remove this one. I'll just switch it to an `await`

Reply:
For your information in the future:
It is indeed a property, but you MUST chain it with an expectation, it can't be used on its own.

https://jestjs.io/docs/en/expect#resolves

Reply:
Ahh, well that explains it. Thought I found a fancy way of using it :laughing: